index.html

```diff
@@ -491,18 +500,11 @@ <h4 class="contact_head">Mubaris NK</h4>
 
   <!-- Custom JavaScript for this theme -->
   <script src="js/scrolling-nav.js"></script>
-
-
-
-
-
-
-  <script src="https://unpkg.com/vue"></script>
-  <script src="js/mentor.js"></script>
-  <script src="js/contestant.js"></script>
-  <script src="js/projects.js"></script>
-  <script src="js/issue-loader.js"></script>
-
+   <script src="https://unpkg.com/vue"></script>
+   <script src="js/mentor.js"></script>
+   <script src="js/contestant.js"></script>
+   <script src="js/projects.js"></script>
```

Comment:
these scripts reference has been repeated ... look below they are already there.

Reply:
That was why I deleted them in the first place, but I get it, your change was just to ass the issue script and not all the ones I deleted.